mathics/core/builtin.py

```diff
@@ -636,7 +653,7 @@ def get_mpmath_function(self, args):
     def eval(self, z, evaluation: Evaluation):
         "%(name)s[z__]"
 
-        args = numerify(z, evaluation).get_sequence()
+        args = cast(Sequence[Number], numerify(z, evaluation).get_sequence())
```

Comment:
Since this just makes silent a warning about something that probably should produce a warning, maybe it worth to add a comment before this line